generators/client/templates/angular/src/main/webapp/app/admin/health/_health.component.html

```diff
@@ -19,8 +19,8 @@
                     <!-- TODO: Add translation for service name -->
                     <td><% if (enableTranslation) { %>{{'health.indicator.' + baseName(health.name) | translate}}<% } else { %><span class="text-capitalize">{{ baseName(health.name) }}</span><% } %> {{subSystemName(health.name)}}</td>
                     <td class="text-xs-center">
-                        <span class="tag" [ngClass]="getTagClass(health.status)" jhi-translate="{{'health.status.' + health.status}}">
-                            {{health.status}}
+                        <span class="tag" [ngClass]="getTagClass(health.status)"<% if (enableTranslation) { %> jhi-translate="{{'health.status.' + health.status}}"<% } else { %> jhi-translate="'health.status.' + health.status"<% } %>>
```

Comment:
Do you mean there is a regex deleting every occurrence of jhi-translate directives  ? Because right now in every template even without i18n the directive is present